mmdet/core/evaluation/eval_hooks.py

```diff
@@ -19,11 +23,35 @@ class EvalHook(Hook):
             If None, whether to evaluate is merely decided by ``interval``.
             Default: None.
         interval (int): Evaluation interval (by epochs). Default: 1.
+        save_best (bool): Whether to save best checkpoint during evaluation.
+            Default: True.
+        key_indicator (str, optional): Key indicator to measure the best
+            checkpoint during evaluation when ``save_best`` is set to True.
+            Options are the evaluation metrics to the test dataset. e.g.,
+            ``bbox_mAP``, ``segm_mAP`` for bbox detection and instance
+            segmentation. ``AR@100`` for proposal recall. If not specified,
+            the first key will be used. Default: None.
+        rule (str, optional): Comparison rule for best score. If set to None,
+            it will infer a reasonable rule. Keys contain 'mAP' or 'AR' will
```

Comment:
Grammatical issues. It should be "Keys containing...". Another problem is that it sounds more natural to change it to "Keys such as 'mAP and 'AR' will be inferred by 'greater' rule. ". If we state "Keys containing...", users may expect a regex matching here.